docs/development_environment.md

```diff
@@ -74,6 +74,18 @@ Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-L
 
 From Windows Store install Ubuntu.
 
+#### Additional steps for Ubuntu 18.04
+
+If you chose Ubuntu 18.04 from the Windows Store some additional steps are needed to install the latest ffmpeg version. When you are promted to acknowledge the displayed information you can press ENTER to proceed.
```

Comment:
```suggestion
If you chose Ubuntu 18.04 from the Windows Store some additional steps are needed to install the latest FFmpeg version. When you are prompted to acknowledge the displayed information you can press ENTER to proceed.
```

Reply:
This is the developer documentation, if feel that this line:

> When you are prompted to acknowledge the displayed information you can press ENTER to proceed.

Might be a bit overkill :)

Reply:
I removed the line. I guess there are not that many people using a complex setup like WSL who would need the additional encouragement to press ENTER. 😅